src/main.cpp

```diff
@@ -4541,7 +4541,7 @@ void static ProcessGetData(CNode* pfrom, const Consensus::Params& consensusParam
                         assert(!"cannot load block from disk");
                     if (inv.type == MSG_CMPCT_BLOCK)
                     {
-                        if (mi->second->nHeight >= chainActive.Height() - 10) {
+                        if (mi->second->nHeight >= chainActive.Height() - MAX_BLOCKS_TO_ANNOUNCE) {
```

Comment:
Right. I wanted to (and think we still should) make it announce using
cmpctblocks even when there are multiple blocks to announce at once (so we
don't lose BIP152's advantages completely in small reorgs), but it turned
out to be harder than I thought.